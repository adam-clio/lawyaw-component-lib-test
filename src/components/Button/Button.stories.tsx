import React from 'react';
import { ComponentMeta, ComponentStory } from '@storybook/react';
import { Button } from './Button';

export default {
  component: Button,
  args: {
    children: 'Im literally a button',
  },
} as ComponentMeta<typeof Button>;

export const Default: ComponentStory<typeof Button> = (args) => (
  <Button {...args} />
);
