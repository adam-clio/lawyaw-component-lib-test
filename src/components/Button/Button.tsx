import * as React from 'react';
export const Button = (props: React.HTMLAttributes<HTMLButtonElement>) => (
  <button {...props} />
);
