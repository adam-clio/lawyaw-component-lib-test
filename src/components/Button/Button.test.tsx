import React from 'react';
import { Button } from './Button';
import { render } from '@testing-library/react';

describe('Button', () => {
  it('should work', () => {
    const rendered = render(<Button />);
    expect(rendered.baseElement.innerHTML).toMatchSnapshot();
  });
});
