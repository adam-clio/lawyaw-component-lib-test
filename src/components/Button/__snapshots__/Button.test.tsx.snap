// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should work 1`] = `"<div><button></button></div>"`;
